foreman/data_refinery_foreman/surveyor/external_source.py

```diff
@@ -111,6 +111,15 @@ def queue_downloader_job_for_original_files(self,
                                                 ):
         """Creates a single DownloaderJob with multiple files to download.
         """
+        source_urls = [original_file.source_url for original_file in original_files]
+        # There is already a downloader job associated with this file.
+        old_assocs = DownloaderJobOriginalFileAssociation.objects.filter(
+            original_file__source_url__in=source_urls)
+        if len(old_assocs) > 0:
+            logger.debug("We found an existing DownloaderJob for these urls.",
```

Comment:
Is this bad enough to warrant a logger.error?

Reply:
Definitely not. My plan is to requeue everything that we have already queued up to make sure that nothing got skipped because of surveyor jobs getting interruptted, so I expect this error to happen once for every sample we've already downloaded, which should be 100's of thousands.